india_compliance/gst_india/constants/custom_fields.py

```diff
@@ -913,7 +913,7 @@
             "label": "e-Invoice Status",
             "fieldtype": "Select",
             "insert_after": "status",
-            "options": "\nPending\nGenerated\nAuto-Retry\nCancelled\nManually Cancelled\nFailed\nNot Applicable",
+            "options": "\nPending\nGenerated\nAuto-Retry\nCancelled\nManually Cancelled\nFailed\nNot Applicable\nPending Cancellation",
             "default": None,
```

Comment:
Can we have a default here as Not Applicable?

Reply:
I think it will issue in patching the data
https://github.com/resilient-tech/india-compliance/blob/19e405d0da25cc98170d3e5da7fb327880523155/india_compliance/patches/post_install/add_einvoice_status_field.py#L18